AssertionError when running tests with babel

FAIL: test_babel (test_babel.TestBabel)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "e:\src\pint\pint\testsuite\test_babel.py", line 19, in test_babel
    "24.0 m�tres"
AssertionError: '24.0 {0} m�tres' != '24.0 m�tres'
- 24.0 {0} m�tres
?     ----
+ 24.0 m�tres

--- a/pint/formatting.py
+++ b/pint/formatting.py
@@ -149,7 +149,8 @@ def formatter(items, as_ratio=True, single_denominator=False,
             for _babel_length in [babel_length] + other_lengths:
                 pat = unit_patterns.get(_key, {}).get(_babel_length, {}).get(plural)
                 if pat is not None:
-                    key = pat.replace('{}', '').strip()
+                    # Don't remove this positional! This is the format used in Babel
+                    key = pat.replace('{0}', '').strip()
                     break
             division_fmt = compound_unit_patterns.get("per", {}).get(babel_length, division_fmt)
             power_fmt = '{}{}'

